airflow/www/decorators.py

```diff
@@ -57,6 +95,10 @@ def wrapper(*args, **kwargs):
                     for k, v in chain(request.values.items(multi=True), request.view_args.items())
                     if k not in fields_skip_logging
                 ]
+                if event and "variable." in event:
```

Comment:
Maybe? (I'm not sure the format of the event var for this)
```suggestion
                if event and event.startswith("variable."):
```

Similar for connection I'd assume.